pypicloud/__init__.py

```diff
@@ -143,6 +143,16 @@ def run_postfork_hooks():
     config.registry.fallback = fallback_mode
     config.registry.always_show_upstream = always_show_upstream
 
+    stream_files = settings.get("pypi.stream_files")
+    if stream_files is None:
+      stream_files = False
+    else:
+      stream_files = asbool(stream_files)
+    config.registry.stream_files = stream_files
```

Comment:
A more concise option:

```suggestion
    config.registry.stream_files = asbool(settings.get("pypi.stream_files", False))
```